### PR TITLE
Update annotations for codeflare-sdk and odh-elyra in the ImageStreams annotations 

### PR DESCRIPTION
**Pull request by atheo89**
Update annotations for codeflare-sdk and odh-elyra in the ImageStreams annotations 

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: manifests/base/jupyter-datascience-notebook-imagestream.yaml [hunk 1]
```diff
@@ -32,10 +32,10 @@ spec:
             {"name": "Pandas", "version": "2.2"},
             {"name": "Scikit-learn", "version": "1.5"},
             {"name": "Scipy", "version": "1.14"},
-            {"name": "Odh-Elyra", "version": "4.1"},
+            {"name": "Odh-Elyra", "version": "4.2"},
             {"name": "PyMongo", "version": "4.8"},
             {"name": "Pyodbc", "version": "5.1"},
-            {"name": "Codeflare-SDK", "version": "0.22"},
+            {"name": "Codeflare-SDK", "version": "0.23"},
```
Comment (jiridanek) [inline manifests/base/jupyter-datascience-notebook-imagestream.yaml:38]: * https://github.com/opendatahub-io/notebooks/pull/757

File: manifests/base/jupyter-datascience-notebook-imagestream.yaml [hunk 2]
```diff
@@ -32,10 +32,10 @@ spec:
             {"name": "Pandas", "version": "2.2"},
             {"name": "Scikit-learn", "version": "1.5"},
             {"name": "Scipy", "version": "1.14"},
-            {"name": "Odh-Elyra", "version": "4.1"},
+            {"name": "Odh-Elyra", "version": "4.2"},
```
Comment (jiridanek) [inline manifests/base/jupyter-datascience-notebook-imagestream.yaml:35]: * https://github.com/opendatahub-io/notebooks/pull/757

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

As these are merged, we can include: 
https://github.com/opendatahub-io/notebooks/pull/765
https://github.com/opendatahub-io/notebooks/pull/757

**Pull request merged**